etc/column_mappings.yaml

```diff
@@ -343,10 +357,29 @@ sample_run_elements:
 
 
 active_runs:
-  - run_id
-  - instrument_id
-  - run_status
-  - created_date
-  - cst_date
-  - project_ids
-  - sample_ids
+    - run_id
+    - instrument_id
+    - run_status
+    - created_date
+    - cst_date
+    - project_ids
+    - sample_ids
```

Comment:
This makes me think that the active runs page has been removed: opened https://github.com/EdinburghGenomics/Reporting-App/issues/202